src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        printLogo();
+        greetUser();
+
+        String userLineInput = "initialize";
+        Task[] taskItems = new Task[100]; //create array to store text entered by user
+        int taskCounter = 0; //track number of tasks
+
+        Scanner in = new Scanner(System.in); //Create scanner object to take in input
+
+        while (!userLineInput.equals("bye")) {
+            userLineInput = in.nextLine(); //user input
+            String[] userWords = userLineInput.split(" ");
+
+            printDottedLine();
+            switch (userWords[0]) {
+            case "list":
+                userInputIsList(taskItems, taskCounter);
+                continue;
+
+            case "todo":
+                userInputIsTodo(userLineInput, taskItems, taskCounter);
+                taskCounter++;
+                printTaskCount(taskCounter);
+                printDottedLine();
+                continue;
+
+            case "event":
+                userInputIsEvent(userLineInput, taskItems, taskCounter);
+                taskCounter++;
+                printTaskCount(taskCounter);
+                printDottedLine();
+                continue;
+
+            case "deadline":
+                userInputIsDeadline(userLineInput, taskItems, taskCounter);
+                taskCounter++;
+                printTaskCount(taskCounter);
+                printDottedLine();
+                continue;
+
+
+            case "done":
+                userInputIsDone(taskItems, taskCounter, userWords[1]);
+                continue;
+
+            case "bye":
+                userInputIsBye();
+                break;
+
+            default:
+            }
+        }
+    }
+
+    private static void userInputIsTodo(String userLineInput, Task[] taskItems, int taskCounter) {
```

Comment:
Try to use more descriptive method names to describe what action the method is executing. Perhaps "userInputIsDo" could be renamed to "addTodo"?